LX/src/main/java/heronarts/lx/LXEngine.java

```diff
@@ -93,6 +93,8 @@
 
   public final LXScriptEngine script;
 
+  public final LXSnapshotController snapshotController;
```

Comment:
There's enough here that I would put everything inside a package heronarts.lx.snapshot. would also call this LXSnapshotEngine for naming parity, and just call this variable "snapshots" so it's lx.engine.snapshots similar to lx.engine.modulation, lx.engine.audio etc.

Reply:
Thanks.  Definitely can use help on LX-style organization and naming conventions.